### PR TITLE
feat: set apiserver tokenRequest flags in 1.20+

### PR DESCRIPTION
**Pull request by jackfrancis**
**Reason for Change**:

This adds soon-to-be-required apiserver config args in anticipation of this upstream PR:

https://github.com/kubernetes/kubernetes/pull/95896

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/defaults-apiserver.go [hunk 1]
```diff
@@ -104,6 +104,11 @@ func (cs *ContainerService) setAPIServerConfig() {
 		}
 	}
 
+	if common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.20.0-alpha.1") {
+		defaultAPIServerConfig["--service-account-issuer"] = "kubernetes.default.svc"
```
Comment (ritazh) [inline pkg/api/defaults-apiserver.go:108]: From https://github.com/kubernetes/kubernetes/pull/95896/files#diff-97e779171708fff85d7b1ee3fac01336bf351c39066428e12526d0fd7da5c05cR201, it seems we also need `service-account-key-file`.

Comment (jackfrancis) [reply pkg/api/defaults-apiserver.go:108]: We actually already pass that config: https://github.com/Azure/aks-engine/blob/master/pkg/api/defaults-apiserver.go#L31

**Review by mboersma — APPROVED** [review]
/lgtm

**Pull request merged**